Unable to run npm run dev from install

1. Did npm install
2. Did npm run dev
```6:49:26 PM - Starting compilation in watch mode...

src/helloworld.ts(1,30): error TS2307: Cannot find module 'skyrimPlatform'.

6:49:26 PM - Found 1 error. Watching for file changes.

```
Steps:
1. 

PS I:\SteamLibrary\steamapps\common\Skyrim Special Edition\Mods\Skyrim Platform\Platform\plugin-example> npm install
npm WARN skyrimplatform-plugin-example@1.0.0 No repository field.

audited 29 packages in 0.462s
found 0 vulnerabilities

PS I:\SteamLibrary\steamapps\common\Skyrim Special Edition\Mods\Skyrim Platform\Platform\plugin-example> 

2. 

(Add config.js with location of Skyrim)

3.  

npm run dev

> skyrimplatform-plugin-example@1.0.0 dev I:\SteamLibrary\steamapps\common\Skyrim Special Edition\Mods\Skyrim Platform\Platform\plugin-example
> node ./tsc/before-first-compilation.js && tsc-watch --onSuccess "node ./tsc/on-success.js"

9:55:01 AM - Starting compilation in watch mode...

src/example.ts(1,47): error TS2307: Cannot find module 'skyrimPlatform'.
src/tests.ts(1,133): error TS2307: Cannot find module 'skyrimPlatform'.
src/tests.ts(3,22): error TS2307: Cannot find module 'skyrimPlatform'.
src/tests.ts(13,12): error TS7006: Parameter 'res' implicitly has an 'any' type.

9:55:02 AM - Found 4 errors. Watching for file changes.
```

Windows 10
npm version: 
```{
  'skyrimplatform-plugin-example': '1.0.0',
  npm: '6.14.15',
  ares: '1.17.2',
  brotli: '1.0.9',
  cldr: '39.0',
  icu: '69.1',
  llhttp: '2.1.3',
  modules: '83',
  napi: '8',
  nghttp2: '1.42.0',
  node: '14.17.6',
  openssl: '1.1.1l',
  tz: '2021a',
  unicode: '13.0',
  uv: '1.41.0',
  v8: '8.4.371.23-node.76',
  zlib: '1.2.11'
}
```
node: v14.17.6
npm: 6.14.15

--- a/skyrim-platform/tools/plugin-example/README.md
+++ b/skyrim-platform/tools/plugin-example/README.md
@@ -24,4 +24,4 @@ Hello SE
 ``,
 
 7. Make sure `src/example.ts` is working. To do this, write `coc riverwood` on the command line. You will be teleported to the location. Talk to Fendal. You should take away 100 gold and the inscription `Thanks for support` will appear. 
-8. Check that hotcode reloading works by changing the debug text in `plugin-example.js` and adding more gold to your inventory (open console and run player.additem f 1000) and talking to Fendal again. Now you should see your changed text in game!
+8. Check that hot code reloading works by changing the debug text in `plugin-example.js` and adding more gold to your inventory (open console and run player.additem f 1000) and talking to Fendal again. Now you should see your changed text in game!

--- a/skyrim-platform/tools/plugin-example/README.md
+++ b/skyrim-platform/tools/plugin-example/README.md
@@ -2,17 +2,22 @@
 
 ## Setting Up
 
-1. Run `npm i` on the command line to install the dependencies.
-2. Create a file `tsc/config.js` with the following contents:
-   ```js
+
+1. Grab the latest version of the Skyrim Platform from https://www.nexusmods.com/skyrimspecialedition/mods/54909?tab=files
+2. Extract the zip and place the files in your Skyrim install folder (Skyrim Special Edition\Data) 
+   ### DO NOT USE A MOD MANAGER AS HOTCODE RELOADING WILL NOT WORK WITH THE VIRTUAL DRIVE.
+3. Copy the Skyrim Special Edition\Data\Platform\Modules\SkyrimPlatform.ts file to the root Skyrim folder Skyrim Special Edition
+3. Go to the Skyrim Special Edition\Data\Platform\plugin-example folder and run `npm i` on the command line to install the dependencies.
+5. Create a file in the Skyrim Special Edition\Data\Platform\plugin-example\tsc folder named `config.js` with the following contents:
+   ```
    module.exports = {
        // Change `seRoot` to the correct path to the Skyrim SE folder. The path should have slashes like this: `/` (not `\\`).
        seRoot: "C:/Program Files (x86)/Steam/steamapps/common/Skyrim Special Edition"
    };
    ```
-
-3. Run `npm run dev`. If everything is ok, the message `Found 0 errors, installing plugin-example.js` will appear.
-4. Log in to Steam and start the game with `skse64_loader.exe`.
+6. 
+7. Run `npm run dev`. If everything is ok, the message `Found 0 errors, installing plugin-example.js` will appear.
+8. Log in to Steam and start the game with `skse64_loader.exe`.
 
 The console should display the following output (console opens to `~` in game).
 ``,
@@ -20,4 +25,5 @@ Hello SE
 [Script] Hello Platform
 ``,
 
-Make sure `src/example.ts` is working. To do this, write `coc riverwood` on the command line. You will be teleported to the location. Talk to Fendal. You should take away 100 gold and the inscription `Thanks for support` will appear. This is a sample script.
+1. Make sure `src/example.ts` is working. To do this, write `coc riverwood` on the command line. You will be teleported to the location. Talk to Fendal. You should take away 100 gold and the inscription `Thanks for support` will appear. 
+2. Check that hotcode reloading works by changing the debug text in plugin-example.js and adding more gold to your inventory (open console and run player.additem f 1000) and talking to Fendal again. Now you should see your changed text in game!

--- a/skyrim-platform/tools/plugin-example/README.md
+++ b/skyrim-platform/tools/plugin-example/README.md
@@ -15,9 +15,8 @@
        seRoot: "C:/Program Files (x86)/Steam/steamapps/common/Skyrim Special Edition"
    };
    ```
-6. 
-7. Run `npm run dev`. If everything is ok, the message `Found 0 errors, installing plugin-example.js` will appear.
-8. Log in to Steam and start the game with `skse64_loader.exe`.
+6. Run `npm run dev`. If everything is ok, the message `Found 0 errors, installing plugin-example.js` will appear.
+7. Log in to Steam and start the game with `skse64_loader.exe`.
 
 The console should display the following output (console opens to `~` in game).
 ``,
@@ -25,5 +24,5 @@ Hello SE
 [Script] Hello Platform
 ``,
 
-1. Make sure `src/example.ts` is working. To do this, write `coc riverwood` on the command line. You will be teleported to the location. Talk to Fendal. You should take away 100 gold and the inscription `Thanks for support` will appear. 
-2. Check that hotcode reloading works by changing the debug text in plugin-example.js and adding more gold to your inventory (open console and run player.additem f 1000) and talking to Fendal again. Now you should see your changed text in game!
+8. Make sure `src/example.ts` is working. To do this, write `coc riverwood` on the command line. You will be teleported to the location. Talk to Fendal. You should take away 100 gold and the inscription `Thanks for support` will appear. 
+9. Check that hotcode reloading works by changing the debug text in plugin-example.js and adding more gold to your inventory (open console and run player.additem f 1000) and talking to Fendal again. Now you should see your changed text in game!

--- a/skyrim-platform/tools/plugin-example/README.md
+++ b/skyrim-platform/tools/plugin-example/README.md
@@ -6,17 +6,16 @@
 1. Grab the latest version of the Skyrim Platform from https://www.nexusmods.com/skyrimspecialedition/mods/54909?tab=files
 2. Extract the zip and place the files in your Skyrim install folder (Skyrim Special Edition\Data) 
    ### DO NOT USE A MOD MANAGER AS HOT CODE RELOADING WILL NOT WORK WITH THE VIRTUAL DRIVE.
-3. Copy the `Skyrim Special Edition\Data\Platform\Modules\SkyrimPlatform.ts` file to the root Skyrim folder Skyrim Special Edition
 3. Go to the `Skyrim Special Edition\Data\Platform\plugin-example` folder and run `npm i` on the command line to install the dependencies.
-5. Create a file in the `Skyrim Special Edition\Data\Platform\plugin-example\tsc` folder named `config.js` with the following contents:
+4. Create a file in the `Skyrim Special Edition\Data\Platform\plugin-example\tsc` folder named `config.js` with the following contents:
    ```
    module.exports = {
        // Change `seRoot` to the correct path to the Skyrim SE folder. The path should have slashes like this: `/` (not `\\`).
        seRoot: "C:/Program Files (x86)/Steam/steamapps/common/Skyrim Special Edition"
    };
    ```
-6. Run `npm run dev`. If everything is ok, the message `Found 0 errors, installing plugin-example.js` will appear.
-7. Log in to Steam and start the game with `skse64_loader.exe`.
+5. Run `npm run dev`. If everything is ok, the message `Found 0 errors, installing plugin-example.js` will appear.
+6. Log in to Steam and start the game with `skse64_loader.exe`.
 
 The console should display the following output (console opens to `~` in game).
 ``,
@@ -24,5 +23,5 @@ Hello SE
 [Script] Hello Platform
 ``,
 
-8. Make sure `src/example.ts` is working. To do this, write `coc riverwood` on the command line. You will be teleported to the location. Talk to Fendal. You should take away 100 gold and the inscription `Thanks for support` will appear. 
-9. Check that hotcode reloading works by changing the debug text in `plugin-example.js` and adding more gold to your inventory (open console and run player.additem f 1000) and talking to Fendal again. Now you should see your changed text in game!
+7. Make sure `src/example.ts` is working. To do this, write `coc riverwood` on the command line. You will be teleported to the location. Talk to Fendal. You should take away 100 gold and the inscription `Thanks for support` will appear. 
+8. Check that hotcode reloading works by changing the debug text in `plugin-example.js` and adding more gold to your inventory (open console and run player.additem f 1000) and talking to Fendal again. Now you should see your changed text in game!

--- a/skyrim-platform/tools/plugin-example/README.md
+++ b/skyrim-platform/tools/plugin-example/README.md
@@ -5,10 +5,10 @@
 
 1. Grab the latest version of the Skyrim Platform from https://www.nexusmods.com/skyrimspecialedition/mods/54909?tab=files
 2. Extract the zip and place the files in your Skyrim install folder (Skyrim Special Edition\Data) 
-   ### DO NOT USE A MOD MANAGER AS HOTCODE RELOADING WILL NOT WORK WITH THE VIRTUAL DRIVE.
-3. Copy the Skyrim Special Edition\Data\Platform\Modules\SkyrimPlatform.ts file to the root Skyrim folder Skyrim Special Edition
-3. Go to the Skyrim Special Edition\Data\Platform\plugin-example folder and run `npm i` on the command line to install the dependencies.
-5. Create a file in the Skyrim Special Edition\Data\Platform\plugin-example\tsc folder named `config.js` with the following contents:
+   ### DO NOT USE A MOD MANAGER AS HOT CODE RELOADING WILL NOT WORK WITH THE VIRTUAL DRIVE.
+3. Copy the `Skyrim Special Edition\Data\Platform\Modules\SkyrimPlatform.ts` file to the root Skyrim folder Skyrim Special Edition
+3. Go to the `Skyrim Special Edition\Data\Platform\plugin-example` folder and run `npm i` on the command line to install the dependencies.
+5. Create a file in the `Skyrim Special Edition\Data\Platform\plugin-example\tsc` folder named `config.js` with the following contents:
    ```
    module.exports = {
        // Change `seRoot` to the correct path to the Skyrim SE folder. The path should have slashes like this: `/` (not `\\`).
@@ -25,4 +25,4 @@ Hello SE
 ``,
 
 8. Make sure `src/example.ts` is working. To do this, write `coc riverwood` on the command line. You will be teleported to the location. Talk to Fendal. You should take away 100 gold and the inscription `Thanks for support` will appear. 
-9. Check that hotcode reloading works by changing the debug text in plugin-example.js and adding more gold to your inventory (open console and run player.additem f 1000) and talking to Fendal again. Now you should see your changed text in game!
+9. Check that hotcode reloading works by changing the debug text in `plugin-example.js` and adding more gold to your inventory (open console and run player.additem f 1000) and talking to Fendal again. Now you should see your changed text in game!

